site/docs/api-quickstart.md

```diff
@@ -108,14 +110,32 @@ To create an Iceberg schema from an existing Avro schema, use converters in `Avr
 
 ```scala
 import org.apache.iceberg.avro.AvroSchemaUtil
+import org.apache.iceberg.Schema
 import org.apache.avro.Schema.Parser
+import org.apache.iceberg.types.Type
+
+val avroSchemaString = """{
```

Comment:
I'm not sure that making this example longer is adding value. This assumes that someone already knows how to use Avro, so we use a short version of a schema to avoid distracting from the more important parts of the example.

Reply:
I find the current examples quite confusing because of the different schema's (Avro/Iceberg), a bit more verbose documentation would have helped me.